app/controllers/api/v0/articles_controller.rb

```diff
@@ -116,6 +118,13 @@ def allowed_to_change_org_id?
             @user.any_admin?
         end
       end
+
+      def validate_article_param_is_hash
+        return if params[:article].respond_to?(:dig)
```

Comment:
I know people aren't big on explicit type checking in Ruby, but `dig` is available arrays, structs, and even `CSV::Row`. If we need a hash, than the explicit `unless params[:article].is_a?(Hash)` seems like the better choice.

Reply:
Great point - I usually go with non-explicit type checking, but this makes a lot of sense! Updated.